pygin/pygin/src/py.string.cpp

```diff
@@ -64,8 +64,8 @@ namespace py
 	{
 	}
 
-	string::string(const wchar_t* Str):
-		string({ Str, Str? wcslen(Str) : 0 })
+	string::string(const wchar_t* Str, size_t Size):
```

Comment:
There are cases where we have `wchar_t*` (returned from FAR API) to be converted to a python string. Using intermediate `std::wstring` (and additional dynamic memory allocations) does not make sense since python API wants `wchar_t*` anyway. Of course, a constructor with `std::wstring` still does make sense for convenience (when you already have `std::wstring`).

Reply:
I mean we already have one that accepts `string_view`.
`string_view` is cheap to construct, so instead of passing `(data, size)` separately we can construct a temporary view instead with `({data, size})`. It won't cause any allocations.

Reply:
Oh. For some reason, I saw this as `wstring` not `string_view`. Reverted.